src/test/java/uk/nhs/digital/nhsconnect/lab/results/inbound/queue/MeshInboundQueueServiceTest.java

```diff
@@ -67,14 +67,14 @@
     @Test
     void receiveInboundMessageIsHandledByInboundQueueConsumerService() throws Exception {
         when(message.getStringProperty(JmsHeaders.CORRELATION_ID)).thenReturn(CORRELATION_ID);
-        when(message.getBody(String.class)).thenReturn("{\"workflowId\":\"LAB_RESULTS_REG\"}");
+        when(message.getBody(String.class)).thenReturn("{\"workflowId\":\"PATH_MEDRPT_V3\"}");
```

Comment:
Could we make the `"{\"workflowId\":\"PATH_MEDRPT_V3\"}"` value into a static variable because it's used in three different tests?

Reply:
Good idea 👍